web/ui/navbar/index.svelte

```diff
@@ -49,28 +51,45 @@
 
 <nav class="navbar">
   <div class="navbar_main" aria-orientation="horizontal" role="menu">
-    {#if $isRefreshing}
+    <div
+      class={`navbar_back-btn ${$side === 'first' ? 'navbar_btn-hidden' : ''}`}
```

Comment:
`is-hidden` modifier should be here instead of 2 elements on the same tag

Reply:
Could you give me an example of valid CSS class name that I can use here? My 'names' is always give an error when I run test:
```bash
Selector .is-hidden does not follow our BEM name system
.is-hidden {
```

Reply:
Because every modifier need to be defined with element.

```css
navbar_back-button.is-hidden {}
```